### DISCUSSION TITLE
BDD 두레 프로젝트 디자인 회고

### DISCUSSION THREAD

**Discussion by mun-kyeong** [BDD-CLUB/challenge-every-week-posting#372] [5. 회고]
[블로그](https://velog.io/@mun-kyeong/%ED%9A%8C%EA%B3%A0-%EB%91%90%EB%A0%88-DooRe-%EB%91%90%EB%A0%88%EC%9D%98-%EB%94%94%EC%9E%90%EC%9D%B8%EC%A0%81-%ED%8A%B9%EC%A7%95-%EC%A0%95%EB%A6%AC)
마지막 BDD에서의 프로젝트 활동 부분의 기록만 남았습니다!
오늘 안에 작성해서 얼른 공동 블로그에 글을 올리고싶네용!! 
파이팅입니다 :)

**Comment by shkisme** [comment]
와,,, 비디디 팀블로그에도 게시하면 좋겠네요 ㅎㅎ
잘 봤습니다~
